whisper.cpp

```diff
@@ -711,6 +711,20 @@ struct whisper_context {
     std::string path_model; // populated by whisper_init_from_file()
 };
 
+static void whisper_default_log(const char * text) {
+    fprintf(stderr, "%s\n", text);
```

Comment:
New lines must be explicitly logged

```suggestion
    fprintf(stderr, "%s", text);
```